<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>1181</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>MetaManage</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1221</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/res/interf.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>1131</width>
       <height>451</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1131</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1131</width>
       <height>451</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>12</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
border-color: rgb(84, 255, 90);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Client</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>1031</width>
         <height>421</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_16">
        <attribute name="title">
         <string>Accueil</string>
        </attribute>
        <widget class="QTableView" name="tableView">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>921</width>
           <height>231</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>310</y>
           <width>141</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Exporter</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonasupprimer">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>320</y>
           <width>111</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>20</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Tirer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>121</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Rechercher</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_CIN_2">
         <property name="geometry">
          <rect>
           <x>780</x>
           <y>310</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 2px solid rgb(0, 0, 0);
    border-radius: 20px;
    color: rgba(255, 255, 255, 0.5); /* Texte avec 50% de transparence */
    padding-left: 20px;
    padding-right: 20px;
    background-color:rgb(0, 0, 127);
}

QLineEdit:hover {
    border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
    border: 2px solid rgb(255, 85, 255);
    background-color: rgb(255, 214, 119);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>ecrire CIN</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>800</x>
           <y>20</y>
           <width>131</width>
           <height>28</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Téléphone</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CIN</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>310</y>
           <width>111</width>
           <height>28</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>PDF</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Ajouter</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>20</y>
           <width>241</width>
           <height>291</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLineEdit" name="lineEdit_CIN">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>211</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid rgb(0, 0, 0);
    border-radius: 20px;
    color: rgba(255, 255, 255, 0.5); /* Texte avec 50% de transparence */
    padding-left: 20px;
    padding-right: 20px;
    background-color:rgb(0, 0, 127);
}

QLineEdit:hover {
    border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
    border: 2px solid rgb(255, 85, 255);
    background-color: rgb(255, 214, 119);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>CIN</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_email">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>211</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid rgb(0, 0, 0);
    border-radius: 20px;
    color: rgba(255, 255, 255, 0.5); /* Texte avec 50% de transparence */
    padding-left: 20px;
    padding-right: 20px;
    background-color:rgb(0, 0, 127);
}

QLineEdit:hover {
    border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
    border: 2px solid rgb(255, 85, 255);
    background-color: rgb(255, 214, 119);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>E-Mail</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_telephone">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>211</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid rgb(0, 0, 0);
    border-radius: 20px;
    color: rgba(255, 255, 255, 0.5); /* Texte avec 50% de transparence */
    padding-left: 20px;
    padding-right: 20px;
    background-color:rgb(0, 0, 127);
}

QLineEdit:hover {
    border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
    border: 2px solid rgb(255, 85, 255);
    background-color: rgb(255, 214, 119);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Téléphone</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_nom">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>211</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid rgb(0, 0, 0);
    border-radius: 20px;
    color: rgba(255, 255, 255, 0.5); /* Texte avec 50% de transparence */
    padding-left: 20px;
    padding-right: 20px;
    background-color:rgb(0, 0, 127);
}

QLineEdit:hover {
    border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
    border: 2px solid rgb(255, 85, 255);
    background-color: rgb(255, 214, 119);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Nom</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_adresse">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>211</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid rgb(0, 0, 0);
    border-radius: 20px;
    color: rgba(255, 255, 255, 0.5); /* Texte avec 50% de transparence */
    padding-left: 20px;
    padding-right: 20px;
    background-color:rgb(0, 0, 127);
}

QLineEdit:hover {
    border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
    border: 2px solid rgb(255, 85, 255);
    background-color: rgb(255, 214, 119);
}
</string>
          </property>
          <property name="text">
           <string>Adresse</string>
          </property>
          <property name="placeholderText">
           <string>Téléphone</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_matricule">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>250</y>
            <width>211</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid rgb(0, 0, 0);
    border-radius: 20px;
    color: rgba(255, 255, 255, 0.5); /* Texte avec 50% de transparence */
    padding-left: 20px;
    padding-right: 20px;
    background-color:rgb(0, 0, 127);
}

QLineEdit:hover {
    border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
    border: 2px solid rgb(255, 85, 255);
    background-color: rgb(255, 214, 119);
}
</string>
          </property>
          <property name="text">
           <string>Matricule</string>
          </property>
          <property name="placeholderText">
           <string>Téléphone</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButtonajouter">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>320</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_modif">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>320</y>
           <width>111</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_17">
        <attribute name="title">
         <string>Statistique</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_18">
        <attribute name="title">
         <string>Mailling/SMS</string>
        </attribute>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Employé</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Fournisseur</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>Produit</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>Livraison</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
